BurnerB/FreeMentors-V2
# #168400598 User create session

#### What does this PR do?
Allows a user to request a session
#### Description of Task to be completed?
* Create the request user session endpoint
* Ensure tests pass
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ft-DB-create-mentorship-session-168400598
* Run npm --init to install all dependancies.
* Run command `npm test` on your terminal
* Tests should pass
* Run the command `npm start` and access post`/api/v1/auth/signup` with valid credentials then post`/api/v1/auth/signin` with valid credentials .
* Using the provided token access the route PATCH`/api/v1/sessions/:mentor_id`
* You should recieve a response body if the data  if successful:
```
{
    "status": 201,
    "data": {
        "id": 76,
        "mentorid": 3,
        "menteeid": 5,
        "questions": "rada rada ni gani?",
        "menteeemail": "johndoe11@email.com",
        "status": "pending",
        "created_on": "2019-09-10T17:20:31.209Z"
    }
}
```
or
```
{
    "status": 403,
    "error": "Mentor cant access this route"
}
```
if the token is a mentor
#### What are the relevant pivotal tracker stories?
[#168457738](https://www.pivotaltracker.com/story/show/168457738)
#### Screenshots (if appropriate) 

![no request twice](https://user-images.githubusercontent.com/45785786/64811208-7b82ca80-d5a5-11e9-8b1c-6df2215a8e01.png)
